### ISSUE TITLE
`runAsGroup` is referenced on explanation but is missing from code example

### ISSUE THREAD

**Issue by rjmco** [kubernetes/website#13535]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**
[security-context.md](https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/configure-pod-container/security-context.md) references a `runAsGroup` parameter which was supposed to exist on the [security-context.yaml](https://github.com/kubernetes/website/blob/master/content/en/examples/pods/security/security-context.yaml) code example. However the code does not include the parameter.

**Proposed Solution:**
Update [security-context.yaml](https://github.com/kubernetes/website/blob/master/content/en/examples/pods/security/security-context.yaml) code example to include the `runAsGroup` parameter.

**Page to Update:**
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

**Comment by doog33k** [comment]
Thanks @rjmco for this bug report, I was wondering If I should create a bug for this ? :+1:

**Comment by rjmco** [comment]
@doog33k I'm not sure if I will be able to answer your question, but from my point of view I thought this issue was considered already a _bug report_ on the website's content. If on the other hand you mean a _bug report_ on Kubernetes' codebase then I don't think that is required.

Finally, I've already created PR #13538 where I've submitted a possible fix to this issue/bug on the website's content.

Let me know if I have clarified your question or I misunderstood it completely.
